client/trino-client/src/main/java/io/trino/client/OkHttpUtil.java

```diff
@@ -282,6 +287,35 @@ private static KeyStore loadTrustStore(File trustStorePath, Optional<String> tru
         return trustStore;
     }
 
+    private static KeyStore loadSystemTrustStore(Optional<String> trustStoreType)
+            throws IOException, GeneralSecurityException
+    {
+        String osName = Optional.ofNullable(StandardSystemProperty.OS_NAME.value()).orElse("");
+        Optional<String> systemTrustStoreType = trustStoreType;
```

Comment:
the `or` method has been introduced since Java 9, the module pom specifies Java 8 as the target JDK:

https://github.com/trinodb/trino/blob/master/client/trino-jdbc/pom.xml#L17

so I don't think we can use that method here